js/back.js

```diff
@@ -65,6 +65,38 @@
 
         return accumulator;
       }, {});
+      if (browsingDataObj.cookies) {
+        // delete cookies here + ignore whitelisted cookies
+        getStorage("cookieWhitelist", function(data) {
+          var domain_list = data.cookieWhitelist
+          getAllCookies({}, function(cookies)
+          {
+            var callbackCount = 0;
+            for (var i = 0; i < cookies.length; i++)
+            {
+              var cookie = cookies[i];
+              var url = getUrl(cookie.domain, cookie.path, cookie.secure);
+
+              var wl = false;
```

Comment:
Suggestion: please use `whitelist` as the variable name, while it's obvious for me that `wl` stands for whitelist, for another person who reads the code, or if I decide to read it after couple of months, I'll need to remember what was this all about.